docker/nsis/start.ps1

```diff
@@ -59,7 +59,7 @@ Do {
 
 }While (-Not $response.result.synchronized)
 
-Write-Progress -Activity "Waiting for monerod sync" -Completed
+Write-Progress -Activity "Waiting for monerod sync" -Completed -status "$percent% Complete:"
```

Comment:
$percent might be not set at this point, but I'll issue a corrective commit after this

Reply:
This was resolved on a14e8996062eeb2027bd4f9edf63db1d72628ba8